src/melee2.cc

```diff
@@ -6975,135 +6966,128 @@ static void process_monster(int m_idx, bool_ is_frien)
 					disturb(0);
 			}
 
-			/* Check for monster trap */
-			if (c_ptr->feat == FEAT_MON_TRAP)
-			{
-				if (mon_hit_trap(m_idx)) return;
-			}
-			else
-			{
```

Comment:
Any chance you could undo the indentation change here? Just keep the surrounding "{" and "}" and keep the indentation as it was.

It's really hard to see if anything important changed here and I fear big conflicts with some upcoming stuff I haven't released yet.